docs/guides/expectations/contributing/how-to-contribute-a-new-expectation-to-great-expectations.md

```diff
@@ -142,79 +144,103 @@ examples = [{
-ColumnMapExpectation class (the parent of your Expectation’s class) does all the work of the second step. It leaves you to define the yes/no question that the Expectation asks about every row in the column. “Questions” are modeled as Metrics in Great Expectations. A Metric is any observable property of data (e.g., numeric stats like mean/median/mode of a column, but also richer properties of data, such as histogram). You can read more about the relationship between Expectations and Metrics in our Core Concepts: Expectations and Metrics.
+* First they ask a yes/no question from every row in that column (that’s the “map”).
+* Then they ask what percentage of rows gave a positive answer to the first question. If the answer to the second question is above a specified threshold (controlled by the mostly argument), the Expectation considers the data valid.
+ 
+`ColumnMapExpectation class` (the parent of your Expectation’s class) does all the work of the second step. It leaves you to define the yes/no question that the Expectation asks about every row in the column. “Questions” are modeled as Metrics in Great Expectations. A Metric is any observable property of data (e.g., numeric stats like mean/median/mode of a column, but also richer properties of data, such as histogram). You can read more about the relationship between Expectations and Metrics in our [Core Concepts: Expectations and Metrics](/docs/reference/metrics).
 
-ExpectColumnValuesToEqualThree class that the template implements declares that the metric that maps each row in the column to the answer to its yes/no question is called column_values.equal_three:
+* `ExpectColumnValuesToEqualThree` class that the template implements declares that the metric that maps each row in the column to the answer to its yes/no question is called `column_values.equal_three`:
 
+````console
 map_metric = "column_values.equal_three"
+````
+
 The parent class expects the variable map_metric to be set. Change the value of map_metric to something that fits your Metric. Follow these two naming conventions:
 
-the name should start with “column_values.”, because it is a “column map” Metric
-the second part of the name (after the “.”) should be in snake_case format
-While many metrics are already implemented within Great Expectations (e.g., column_values.match_regex, column_values.json_parseable, etc.), column_values.equal_three is not. You will define and implement this new Metric.
+	* the name should start with “column_values.”, because it is a “column map” Metric
+	* the second part of the name (after the “.”) should be in snake_case format
+
+* While many metrics are already implemented within Great Expectations (e.g., column_values.match_regex, column_values.json_parseable, etc.), column_values.equal_three is not. You will define and implement this new Metric.
```

Comment:
<img width="963" alt="Screen Shot 2021-07-09 at 5 09 03 PM" src="https://user-images.githubusercontent.com/49923762/125136211-5d6e4e80-e0d8-11eb-80b2-a4ed2b824737.png">
<img width="759" alt="Screen Shot 2021-07-09 at 5 08 23 PM" src="https://user-images.githubusercontent.com/49923762/125136169-462f6100-e0d8-11eb-81c6-dfa7f36d4861.png">

All the text under this should be nested per RTD

Reply:
Code blocks for the different column_values would be great as well!